src/lib/routes/admin-api/project/api-token.ts

```diff
@@ -116,8 +125,12 @@ export class ProjectApiTokenController extends Controller {
                 openApiService.validPath({
                     tags: ['Projects'],
                     operationId: 'deleteProjectApiToken',
+                    summary: 'Delete a project API token.',
+                    description:
+                        'Endpoint that allows deletion of a project API token.',
                     responses: {
                         200: emptyResponse,
+                        ...getStandardResponses(401, 403),
```

Comment:
This one actually seems to hang in tests when given a wrong project/token

Reply:
I'm unsure if that's just in my unit tests or if this is something that affects it during runtime as well. I'll add a task